app/views/submissions/_assignment_files_info.html.erb

```diff
@@ -0,0 +1,26 @@
+<div class='pane-wrapper'>
+  <div class='pane' style='margin-bottom: 1em;'>
+    <% if assignment.assignment_files.length == 0 %>
+      <p><%= t('assignments.no_required_files') %></p>
```

Comment:
You can remove this part (it might be a bit misleading). Instead, just show the required files list if there's at least one required file. you can use `assignment.assignment_files.exists?` for this check.